cmd/cmd.go

```diff
@@ -902,6 +934,26 @@ func generateInteractive(cmd *cobra.Command, opts generateOptions) error {
 
 		if len(prompt) > 0 && multiline == MultilineNone {
```

Comment:
I think you can abstract this if conditions to a function so it can be easy to test. nice work btw.